plugins/hbase_logs.yaml

```diff
@@ -0,0 +1,132 @@
+version: 0.0.1
+title: Apache HBase
+description: Log parser for Apache HBase
+parameters:
+  - name: enable_master_log
+    type: bool
+    default: true
+  - name: master_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-master-*.log"
+  - name: enable_region_log
+    type: bool
+    default: true
+  - name: region_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-regionserver-*.log"
+  - name: enable_zookeeper_log
+    type: bool
+    default: false
+  - name: zookeeper_log_path
+    type: "[]string"
+    default: 
+    - "/usr/local/hbase*/logs/hbase*-zookeeper-*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_master_log }}
+    filelog/master:
+      include:
+        {{ range $fp := .master_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+          line_start_pattern: '^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+|^\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}'
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          if:  '$record matches "^\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2},\\d{3}\\s+"'
```

Comment:
I think all these `$record`s should be `body` or just `$`; I don't think $record is valid in OTel expressions to reference the log message.

Reply:
Just went in and switched that